python/django_bolt/openapi/schema_generator.py

```
@@ -1258,7 +1276,7 @@ def _type_to_schema(self, type_annotation: Any, register_component: bool = False
         # Default to generic object
         return Schema(type="object")
 
-    def _struct_to_schema(self, struct_type: type) -> Schema:
+    def _struct_to_schema(self, struct_type: type, register_component: bool = False) -> Schema:
```

Ruff (0.15.18)

[warning] Boolean-typed positional argument in function definition

(FBT001)